railties/lib/rails/application/configuration.rb

```diff
@@ -278,22 +278,30 @@ def load_defaults(target_version)
         @loaded_config_version = target_version
       end
 
-      def encoding=(value)
+      # :attr_writer: encoding
+
+      # Documentated as an attr_writer
+      def encoding=(value) # :nodoc:
         @encoding = value
         silence_warnings do
           Encoding.default_external = value
           Encoding.default_internal = value
         end
       end
 
+      # Configures the application to be API only. Sets the debugging
+      # information rendering to preserve the response format.
       def api_only=(value)
         @api_only = value
         generators.api_only = value
 
         @debug_exception_response_format ||= :api
       end
 
-      def debug_exception_response_format
+      # :attr_accessor: debug_exception_response_format
```

Comment:
Should this be `:attr_reader:`?

```suggestion
      # :attr_reader: debug_exception_response_format
```